SConstruct

```diff
@@ -1,3 +1,59 @@
 # -*- python -*-
-from lsst.sconsUtils import scripts
-scripts.BasicSConstruct("ap_pipe", disableCc=True)
+import os
+
+from lsst.sconsUtils import scripts, targets
+from lsst.sconsUtils.state import env
+from lsst.sconsUtils.utils import libraryLoaderEnvironment
+from SCons.Script import Default
+
+# Python-only package
+# Force shebang and policy to come first so the file first appears in the bin
+# directory before it is used. This is required to run on macos.
+PKG_ROOT = env.ProductDir("ap_pipe")
+
+additional_fakes_tasks = os.path.join(
+    PKG_ROOT,
+    "pipelines",
+    "_ingredients",
+    "injection",
+    "ApPipePostInjectedTasks.yaml",
+)
+ap_pipe_with_fakes_path = os.path.join(
+    PKG_ROOT,
+    "pipelines",
+    "_ingredients",
+    "ApPipeWithFakes.yaml"
+)
+
+subset_names = ["apPipe", "prompt"]
+
+ingredients_ap_pipe_with_fakes = env.Command(
+    target=ap_pipe_with_fakes_path,
+    source=os.path.join(PKG_ROOT, "pipelines", "_ingredients", "ApPipe.yaml"),
+    action=" ".join(
+        [
+            libraryLoaderEnvironment(),
+            f"make_injection_pipeline  -t preliminary_visit_image -r $SOURCE -f $TARGET "
+            f"-a {additional_fakes_tasks} -s "+" -s ".join(subset_names),
```

Comment:
Should `-s` be used twice here?

Reply:
And if `subset_names` is length zero, this still prints a single `-s` then? Is that an issue?

Reply:
Adopted! Thanks